Library/Formula/apcupsd.rb

```diff
@@ -0,0 +1,107 @@
+class Apcupsd < Formula
+  desc "Daemon for controlling APC UPSes"
+  homepage "http://www.apcupsd.org"
+  url "https://downloads.sourceforge.net/project/apcupsd/apcupsd%20-%20Stable/3.14.13/apcupsd-3.14.13.tar.gz"
+  sha256 "57ecbde01d0448bf8c4dbfe0ad016724ae66ab98adf2de955bf2be553c5d03f9"
+
+  option "with-all", "Enable all optional modules"
+  option "with-cgi", "Compile CGI programs"
+  option "without-apcsmart", "No APC SmartUPS serial driver"
+  option "without-dumb", "No dumb UPS driver"
+  option "with-usb", "Compile USB UPS code"
+  option "without-net", "No network driver for slaves"
+  option "without-snmp", "No SNMP driver"
+  option "with-test", "Compile TEST driver code"
+  option "without-pcnet", "No PCNET driver"
+  option "with-modbus-usb", "Compile MODBUS/USB driver code"
+  option "without-modbus", "No MODBUS driver"
+  option "with-apcagent", "Build menubar app for Mac OS X"
+  option "with-install-distdir", "Install distribution"
+
+  depends_on "libusb-compat"
+
+  if build.with?("cgi")
```

Comment:
Technically speaking, `depends_on "gd" if build.with? "cgi"` is fine as well :wink: